test/nexus_test.go

```diff
@@ -960,6 +1047,77 @@ func TestWorkflowTestSuite_WorkflowRunOperation_WithCancel(t *testing.T) {
 	}
 }
 
+func TestWorkflowTestSuite_NexusSyncOperation_ScheduleToCloseTimeout(t *testing.T) {
+	sleepDuration := 500 * time.Millisecond
+	op := temporalnexus.NewSyncOperation(
```

Comment:
A sync operation should also have a task timeout of 10s right? Are we enforcing that?

Reply:
No. Good call, we should set the `Request-Timeout` header here as well.
I'll submit a PR for that.

Reply:
https://github.com/temporalio/sdk-go/pull/1653